src/test/java/api/requests/RequestsAPILocationTests.java

```diff
@@ -52,27 +51,21 @@ public void locationIsIncludedForSingleRequest()
       .forItem(smallAngryPlanet)
       .by(requester));
 
-    JsonObject representation = request.getJson();
-
-    assertThat("has item location",
-      representation.getJsonObject("item").containsKey("location"), is(true));
+    JsonObject createdRequest = request.getJson();
 
-    assertThat(
-      representation.getJsonObject("item").getJsonObject("location").getString("name"),
-      is("2nd Floor - Economics"));
+    assertThat(createdRequest, hasItemLocationProperties("Djanogly Learning Resource Centre", "NU/JC/DL/2FE"));
 
     Response fetchedRequestResponse = requestsClient.getById(request.getId());
 
     assertThat(fetchedRequestResponse.getStatusCode(), is(200));
 
-    JsonObject fetchedLoan = fetchedRequestResponse.getJson();
+    JsonObject fetchRequest = fetchedRequestResponse.getJson();
 
     assertThat("has item location",
-      fetchedLoan.getJsonObject("item").containsKey("location"), is(true));
+      fetchRequest.getJsonObject("item").containsKey("location"), is(true));
+
+    assertThat(fetchRequest, hasItemLocationProperties("Djanogly Learning Resource Centre", "NU/JC/DL/2FE"));
```

Comment:
Thank you for extracting this into a custom matcher, I like this approach.

Could this matcher benefit from using composition of matchers, similar to the `ValidationErrorMatchers`?

Reply:
It could. Done.

Reply:
Thank you. I personally find this more explicit and easier to read, I hope you do too.